docs/FEATURES.md

```diff
@@ -0,0 +1,28 @@
+# Features offered by IPFS-Desktop
+
+This document is intended to capture the list of features offered by IPFS-Desktop, whether they're fully done, or in-progress.
+
+## Legend
+- :grey_question: : Status unknown
+- 🥚 : In design phase
+- 🐣 : Work in progress
+- 🐥 : Try it out (implementation exists, but requires additional work or setup)
+- 🐓 : Shipped (works out of the box when ipfs-desktop is installed)
+
+
+## Native OS features
+
+| Feature | Linux/BSD | macOS | Windows |
+| ---- | ---- | ---- | ---- |
+| System Tray/Menubar Icon | 🐓 |🐓 | 🐓 |
+| [Automatic updates](https://github.com/ipfs-shipyard/ipfs-desktop/issues/668) | 🐥 | 🐓 | 🐓 |
+| [System-wide protocol handler](https://github.com/ipfs-shipyard/ipfs-desktop/issues/807)  | 🥚   | 🐓   | 🐓  |
+| [Dropbox-like mounting of MFS](https://github.com/ipfs-shipyard/ipfs-desktop/issues/618) | 🐣    | 🐣   | 🐣  |
+| ["Add to IPFS" from Native File Manager](https://github.com/ipfs-shipyard/ipfs-desktop/issues/678)  |  :grey_question:    |  :grey_question:    | 🐓 |
+| [Add 'ipfs' to PATH](https://github.com/ipfs-shipyard/ipfs-desktop/issues/727) | 🐓 | 🐓 | 🐓 |
```

Comment:
`PATH` was removed in https://github.com/ipfs/ipfs-desktop/pull/2140, so its back to egg stage:

```suggestion
| [Add 'ipfs' to PATH](https://github.com/ipfs-shipyard/ipfs-desktop/issues/727) | 🥚  | 🥚 | 🥚 |
```